index.js

```diff
@@ -11,6 +11,9 @@ var typeColors = {
 var defaultOptions = {
   indent: '  ',
   newLine: '\n',
+  printVar: function printVar (variable) {
+    return JSON.stringify(variable);
+  },
```

Comment:
Why does this only override `typeof variable === 'object' || typeof variable === 'string') && !(variable instanceof RegExp)`? Would it make sense to change all variables?